build-new-version.sh

```diff
@@ -215,12 +216,22 @@ EOF
     }
     return get_hash(HASHES, triple)
 
+def cargo_url(triple):
+    URLS = {
+EOF
+    for triple in $RUSTC_TRIPLES; do
+        echo >>${CARGO_BIN_RECIPE} "        \"$triple\": \"$(cargo_url $triple)\","
+    done
+
+    cat <<EOF >>${CARGO_BIN_RECIPE}
+    }
+    return get_hash(URLS, triple)
```

Comment:
Not a big thing but if we are going to regenerate at all again, it might be worth changing `get_hash` to `get_by_triple` (still nice to have for generate a better error message).